tests/gold_tests/autest-site/setup.cli.ext

```diff
@@ -123,22 +122,13 @@ if ENV['ATS_BIN'] is not None:
     Variables.trafficserver_version = out
     host.WriteVerbose(['ats'], "Traffic server version:", out)
 
-    # this queries tsxs for build flags so we can build code for the tests and get certain
-    # useful flags as which openssl to use when we don't use the system version
+    # ToDo: This used to use tsxs to find some libraries, maybe it needs to use cmake tools?
```

Comment:
Let's just remove this TODO. Now that our tools are just built with cmake itself, I think this isn't applicable anymore.

The rest of this PR looks fine. Poke me when you update and I'll approved.